docker/Dockerfile.scope

```diff
@@ -1,4 +1,4 @@
-FROM weaveworks/cloud-agent
+FROM openebs/cloud-agent
```

Comment:
I think we also don't need this change. Instead let it build and tag the final image with openebs prefix